### PR TITLE
#2 Registration Page

### PR DESCRIPTION
**Pull request by lithobraking**

### CODE REVIEW

File: src/components/LandingPage/LandingPage.js [hunk 1]
```diff
@@ -1,4 +1,6 @@
 import React from 'react';
+import { Link } from 'react-router-dom';
```
Comment (thirionlogan) [inline src/components/LandingPage/LandingPage.js:2]: `import { Link as RouterLink} from 'react-router-dom'; `

File: src/components/LandingPage/LandingPage.js [hunk 2]
```diff
@@ -60,9 +62,11 @@ function LandingPage(props) {
           NuLeaf
         </Typography>
         <Container maxWidth='xs' className={classes.buttonContainer}>
-          <Button variant='contained' color='primary'>
-            Sign up today
-          </Button>
+          <Link to="/register">
+            <Button  variant='contained' color='primary'>
```
Comment (thirionlogan) [inline src/components/LandingPage/LandingPage.js:66]: intead of wrapping Button in Link, ` <Button  variant='contained' color='primary' to='/register' component={RouterLink}>`

File: src/components/RegistrationPage/RegistrationPage.js [hunk 3]
```diff
@@ -0,0 +1,141 @@
...
+      {'Copyright © '}
+      <Link color="inherit" href="https://material-ui.com/">
+        NuLeaf
+      </Link>{' '}
+      {new Date().getFullYear()}
+      {'.'}
+    </Typography>
+  );
+}
+
```
Comment (thirionlogan) [inline src/components/RegistrationPage/RegistrationPage.js:28]: Eventually, this should be abstracted to its own component

File: src/components/RegistrationPage/RegistrationPage.js [hunk 4]
```diff
@@ -0,0 +1,141 @@
...
+    margin: theme.spacing(3, 0, 2),
+  },
+}));
+
+export default function RegistrationPage() {
+  const classes = useStyles();
+
+  return (
+    <Container component="main" maxWidth="xs">
+      <CssBaseline />
```
Comment (thirionlogan) [inline src/components/RegistrationPage/RegistrationPage.js:54]: This is redundant if there is already one in App.js

**Review by adder206 — COMMENTED** [review]
Henry reviewed the code along with Logan during a shared session.

**Review by thirionlogan — CHANGES_REQUESTED** [review]
see comments

**Review by adder206 — CHANGES_REQUESTED** [review]
Parker to make suggested changes prior to merging

**Review by adder206 — COMMENTED** [review]
update href away from YouTube

**Review by thirionlogan — DISMISSED** [review]

**Review by thirionlogan — APPROVED** [review]

**Review by adder206 — APPROVED** [review]

**Pull request merged**